kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -129,19 +127,19 @@ download_akse()
         exit 1
     fi
     
-    DEFINITION_TEMPLATE_NAME=azurestack-kubernetes$K8S_AZURE_CLOUDPROVIDER_VERSION.json   
-    TEMPLATE_URL="https://raw.githubusercontent.com/$AKSE_REPO/$AKSE_BRANCH/examples/azurestack/$DEFINITION_TEMPLATE_NAME"
+    DEFINITION_TEMPLATE_NAME="kubernetes-azurestack-azure-ad.json"
```

Comment:
Can we check if we can rename or add another file with generic name like: kubernetes-azurestack.json in master as well as in the released patch ?.